.env.dist

```diff
@@ -7,7 +7,7 @@ PUBLIC_PORT_DUMMY_API=8083
 PUBLIC_PORT_S3=8084
 PUBLIC_PORT_JATS_INGESTER=8085
 # TODO: put this to latest as a default, when `latest` is available
-REVISION_BROWSER=75771f41a162de38bc652f3d85d8531805a2df42
+REVISION_BROWSER=ae44b4d7b3a51f4b829ef3cde85fd83eeb7fcb7f
```

Comment:
added `browser` `Git submodules` to Dependabot to update `scripts` and eventually get a `latest` to put here